__tests__/__snapshots__/main.test.ts.snap

```diff
@@ -3,37 +3,37 @@
 exports[`Shows stats when files are added 1`] = `
 "**Total**
 
-Files count | Type | Total bundle size | % Changed
```

Comment:
type was mostly just excessive, so inlining the gzip when necessary, saving on characters